kafka-ui-react-app/src/components/common/Input/__tests__/Input.spec.tsx

```diff
@@ -26,7 +26,7 @@ describe('Custom Input', () => {
           rightIcon: 'fas fa-address-book',
         })
       );
-      expect(component.baseElement).toMatchSnapshot();
+      expect(component.baseElement).toBeInTheDocument();
```

Comment:
screen

Reply:
I can't see any possibility to find icons using screen